server/handler.go

```diff
@@ -904,6 +905,96 @@ func (h *Handler) SetStoreLimitTTL(data string, value float64, ttl time.Duration
 	}, ttl)
 }
 
+// IsLeader return ture if this server is leader
+func (h *Handler) IsLeader() bool {
+	return h.s.member.IsLeader()
+}
+
+// PackHistoryHotReadRegions get read hot region info in HistoryHotRegion form.
+func (h *Handler) PackHistoryHotReadRegions() (historyHotRegions []core.HistoryHotRegion, err error) {
+	hotReadLeaderRegions := h.GetHotReadRegions().AsLeader
+	historyLeaderHotRegions, err := h.packHotRegions(hotReadLeaderRegions, true, core.HotRegionTypes[0])
+	if err != nil {
+		return
+	}
+	hotReadPeerRegions := h.GetHotReadRegions().AsPeer
+	historyPeerHotRegions, err := h.packHotRegions(hotReadPeerRegions, false, core.HotRegionTypes[0])
+	if err != nil {
+		return
+	}
+	historyHotRegions = append(historyHotRegions, historyLeaderHotRegions...)
```

Comment:
Maybe only historyPeerHotRegions is enough, we can judge it according to the `is_leader` field.  it can reduce storage size, and